### PR TITLE
Adding fallback support for pyyaml.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
Reviewing now.

File: roles/lib_utils/library/repoquery.py [hunk 1]
```diff
@@ -29,13 +29,18 @@
...
 import re  # noqa: F401
-# pylint: disable=import-error
-import ruamel.yaml as yaml  # noqa: F401
 import shutil  # noqa: F401
 
+# pylint: disable=import-error
+try:
+    import ruamel.yaml as yaml  # noqa: F401
+except ImportError:
+    import yaml
```
Comment (ashcrow) [inline roles/lib_utils/library/repoquery.py]: :+1:

File: roles/lib_openshift/library/oadm_manage_node.py [hunk 2]
```diff
@@ -374,7 +378,10 @@ def load(self, content_type='yaml'):
         # check if it is yaml
         try:
             if content_type == 'yaml' and contents:
-                self.yaml_dict = yaml.load(contents, yaml.RoundTripLoader)
+                if hasattr(yaml, 'RoundTripLoader'):
+                    self.yaml_dict = yaml.load(contents, yaml.RoundTripLoader)
+                else:
+                    self.yaml_dict = yaml.load(contents)
```
Comment (ashcrow) [inline roles/lib_openshift/library/oadm_manage_node.py]: nit: ``safe_load``. Won't block on it though.

File: roles/lib_openshift/library/oadm_manage_node.py [hunk 3]
```diff
@@ -334,7 +334,11 @@ def write(self):
         if hasattr(self.yaml_dict, 'fa'):
             self.yaml_dict.fa.set_block_style()
 
-        Yedit._write(self.filename, yaml.dump(self.yaml_dict, Dumper=yaml.RoundTripDumper))
+        # pylint: disable=no-member
+        if hasattr(yaml, 'RoundTripDumper'):
```
Comment (ashcrow) [inline roles/lib_openshift/library/oadm_manage_node.py]: Currently these modules all import ``ruamel.yaml`` directly and will always have ``yaml.RoundTripDumper``. Will these be updated to ``try/except`` an ``ImportError`` like reqoquery?

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
@kwoodson :+1: to the changes. Assuming CI passes I say LGTM.

**Comment by chrrrles** [comment]
Thanks @kwoodson  - @sdodson  - Do you feel that the workaround in #3383 will be present in future versions of openshift-ansible?

**Pull request merged**